app/frontend/src/features/dashboard/constants.ts

```diff
@@ -47,7 +47,7 @@ The events are held every week on:
 
 The events are held on Zoom at this link: [https://couchers.org/weekly-social](https://couchers.org/weekly-social).
 
-More information can be found on the [forums](https://community.couchers.org/)!`;
+More event information can be found [here](https://community.couchers.org/c/announcements/events/23)!`;
```

Comment:
```suggestion
You can find [more event information here](https://community.couchers.org/c/announcements/events/23)!`;
```

This way is more accessible - people that use screen readers often flick through links by themselves, so links should contain information about where they are going.